### PR TITLE
ui: ui interface for topology rules

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

File: statics/js/components/topology.js [hunk 1]
```diff
@@ -164,6 +164,9 @@ var TopologyComponent = {
           <tab-pane title="Workflows">\
             <workflow-call></workflow-call>\
           </tab-pane>\
+          <tab-pane title="TopologyRules" v-if="topologyMode === \'live\'">\
```
Comment (lebauce) [inline statics/js/components/topology.js]: title="TopologyRules"
=>
title="Topology rules"
or title="Topology"

File: statics/js/components/noderule-list.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        </i>\
+        {{noderule.UUID}}\
+      </div>\
+      <dl class="dl-horizontal">\
+        <dt v-if="noderule.Name">Name</dt>\
+        <dd v-if="noderule.Name">{{noderule.Name}}</dd>\
+        <dt v-if="noderule.Description">Description</dt>\
+        <dd v-if="noderule.Description">{{noderule.Description}}</dd>\
+        <dt v-if="noderule.Query">Query</dt>\
+        <dd v-if="noderule.Query">{{noderule.Query}}</dd>\
```
Comment (lebauce) [inline statics/js/components/noderule-list.js]: You should probably display the type.

Comment (masco-enovance) [reply statics/js/components/noderule-list.js]: added Action & Type

File: statics/js/components/topology.js (hunk 1 above)
Comment (masco-enovance) [reply statics/js/components/topology.js]: done

**Comment by masco-enovance** [comment]
run skydive-cdd-overview-tests

**Pull request merged**